pkg/windows/connectivity.go

```diff
@@ -82,11 +82,12 @@ func (c *sshConnectivity) init() error {
 			ssh.PublicKeys(c.signer),
 		},
 		HostKeyCallback: ssh.InsecureIgnoreHostKey(),
+		Timeout:         retry.SSHConnTimeout,
```

Comment:
Before there was no SSH connection timeout; given the default is zero; The retry was handled only by retry.Timeout.

Will the two newly introduced timeouts overlap or aggregate?

Reply:
They will overlap. So overall the time table looks like this
| Time  |  Action |
|---|---|
| 0s  | dial attempt 1  |
| 45s  | timeout 1  |
| 1m  | dial attempt 2  |
| 1m 45s  | timeout 2  |
| 2m  | dial attempt 3  |
| 2m 45s  | timeout 3  |
| 3m  | dial attempt 4  |
| 3m 45s  | timeout 4  |
| 4m  | dial attempt 5  |
| 4m 45s  | timeout 5  |
| 5m  | throw reconcile error  |